.gitignore

```diff
@@ -2,5 +2,6 @@
 /.build
 /Packages
 /*.xcodeproj
-.swiftpm
-.vscode/
+/.swiftpm
+/.vscode
+/Package.resolved
```

Comment:
Unless there's a pressing reason to remove it, let's bring back `Package.resolved`. Github's dependabot relies on it to scan for vulnerabilities in dependencies and that automatic check is nice to have.